src/clj/rems/api/catalogue.clj

```diff
@@ -2,19 +2,24 @@
   (:require [compojure.api.sweet :refer :all]
             [rems.api.schema :refer :all]
             [rems.api.util] ; required for route :roles
+            [rems.roles :as roles]
+            [rems.auth.util :refer [throw-forbidden]]
             [rems.api.services.catalogue :as catalogue]
             [ring.util.http-response :refer :all]
-            [schema.core :as s]))
+            [schema.core :as s]
+            [rems.config :refer [env]]
+            [rems.api.util :refer [check-roles]]))
```

Comment:
You should sort the dependencies.

Reply:
not sure how?

Reply:
alphabetically by namespace, e.g. `rems.api.util` goes before `rems.config`

Reply:
right, I can do it

Reply:
Yes, it makes for a slightly more logical grouping.